docs/source/en/conceptual/contribution.mdx

```diff
@@ -12,64 +12,87 @@ specific language governing permissions and limitations under the License.
-the problems they encounter. So thank you for reporting an issue.
+    Remember, GitHub issues are reserved for technical questions directly related to the `diffusers` library, 	bug reports, feature requests, or feedback on the library design.
 
-First, we would really appreciate it if you could **make sure the bug was not
-already reported** (use the search bar on GitHub under Issues).
+	In a nutshell, this means that everything that is **not** related to the **code of the diffusers library** should **not** be asked on GitHub, but rather on either the [forum](TODO:PVP) or [discord](TODO:PVP) as explained in [1.](TODO:PVP).
 
-### Do you want to implement a new diffusion pipeline / diffusion model?
+	**Please consider the following guidelines when open a new issue**:
+	- Make sure you have searched whether your issue has already been asked before (use the search bar on GitHub under Issues).
+	- Please never report a new issue on another (related) issue. If another issue is highly related, please
+	open a new issue nevertheless and link to the related issue.
+	- Make sure your issue is written in English. Plesae use on of the great, free online trasnlation services, such as [DeepL](TODO:PVP) to translate from your native language to English if you are not conformtible in English.
+	- Check whether your issue might be solved by updating to the newest `diffusers` version. Before posting your issue, please make sure that `python -c "import diffusers; print(diffusers.__version__)"` is higher or matches the latest `diffusers` version.
+	- Remember that the more effort you put into opening a new issue, the higher the quality of your answer will be and the better the overall quality of the `diffusers` issues.
 
-Awesome! Please provide the following information:
+New issues usually include the following.
 
-* Short description of the diffusion pipeline and link to the paper;
-* Link to the implementation if it is open-source;
-* Link to the model weights if they are available.
+#### 2.1. Reproducible, minimal bug reports.
+
+A bug report should always have a reproducible code snippet and be as minimal and concise as possible.
+This means in more detail:
+- Narrow the bug down as much as you can, **do not just dump your whole code file**
+- Format your code
+- Do not include any external libraries except for `diffusers` depends on them.
+- **Always** provide all necessary information about your environment; for this you can run: `diffusers-cli env` in your shell and copy-paste the displayed information to the issue.
+- Explain the issue. If the reader doesn't know what the issue is and why it is an issue, she cannot solve it.
+- **Always** make sure the reader can reproduce your issue with as little effort as possible. If you code snippet cannot be run because of missing libraries or undefined variables, the reader cannot help you.  Make sure your reproducible code snippet is as minimal as possible and can be copy-pasted into a simple Python shell.
```

Comment:
```suggestion
- **Always** make sure the reader can reproduce your issue with as little effort as possible. If your code snippet cannot be run because of missing libraries or undefined variables, the reader cannot help you.  Make sure your reproducible code snippet is as minimal as possible and can be copy-pasted into a simple Python shell.
```